tests/resizable_fdt.c

```diff
@@ -0,0 +1,20 @@
+#include <file.h>
+#include <filedesc.h>
+#include <ktest.h>
+
+static int test_resizable_fdt() {
+  fdtab_t *fdt_test = fdtab_alloc();
+  file_init();
```

Comment:
Why do you need to call `file_init` here? It should have been already called by `kernel_init`.